plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveStorageFormat.java

```diff
@@ -61,57 +61,70 @@ public enum HiveStorageFormat
     ORC(
             ORC_SERDE_CLASS,
             ORC_INPUT_FORMAT_CLASS,
-            ORC_OUTPUT_FORMAT_CLASS),
+            ORC_OUTPUT_FORMAT_CLASS,
+            ".orc"),
```

Comment:
can we extract these to constants? or an enum? can we omit dot and add it where we use this extension?

Reply:
I disagree with extracting as constants. We must jump to constants to see the value.